File: cs/Iris/ParserUtils/LexParseException/UnknownField.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    class UnknownField : BaseException
    {
        public string fieldName;
        public string charName;

        public UnknownField(int lNum, int cNum, string cName, string field)
            : base(lNum, cNum)
        {
            charName = cName;
            fieldName = field;
        }

        public override string ExceptionToString()
        {
            StringBuilder str = new StringBuilder();

            str.Append("unknown field '");
            str.Append(fieldName);
            str.Append("' in ");
            str.Append(charName);
            str.Append(base.stringTrace());

            return str.ToString();
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/UnknownField.java
package com.rokuan.iris.exception;

public class UnknownField extends BaseException
{
	public String fieldName;
	public String charName;

	public UnknownField(int lNum, int cNum, String cName, String field)
	{
		super(lNum, cNum);
		charName = cName;
		fieldName = field;
	}

	public String ExceptionToString()
	{
		StringBuilder str = new StringBuilder();

		str.append("unknown field '");
		str.append(fieldName);
		str.append("' in ");
		str.append(charName);
		str.append(super.stringTrace());

		return str.toString();
	}
}
